Xamarin.Forms.Platform.WPF/Properties/AssemblyInfo.cs

```diff
@@ -71,4 +72,6 @@
 
 // Others
 [assembly: Xamarin.Forms.Dependency(typeof(ResourcesProvider))]
-[assembly: Xamarin.Forms.Dependency(typeof(Deserializer))]
\ No newline at end of file
+[assembly: Xamarin.Forms.Dependency(typeof(Deserializer))]
+
+[assembly: AssemblyVersion("2.0.0.0")]
```

Comment:
We shouldn't need both. The `GenerateAssemblyInfo` should automatically include this attribute. I get a compiler error with it...